docs/ums/README.md

```diff
@@ -200,15 +305,43 @@ curl
 ### Using the Automation Module
 You need to set up automation which will catch the `UMS: password reset request` event. Then, when you initiate a reset password, the user will get an email with a pre-made template message ([see Automation](/automation)). Inside you should create a link to a page with a new password entry form. From this page you can make a call `resetPassword` with a token from URL, thjs will allow Jexia to handle the request and apply changes to the new user to enable a new password.     
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
  
 ```js
 // To request email with new token: 
-ums.requestResetPassword('Elon@tesla.com');
+ums
+.requestResetPassword('Elon@tesla.com')
+.subscribe(user => {}, error=>{});   
 
 // To apply newpassword
-ums.resetPassword(Token, newPassword);
+ums
+.resetPassword(Token, newPassword)
+.subscribe(user => {}, error=>{});   
+```
+
+</template>
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
next release.